src/client/pages/VergunningDetail/StatusLineItems.tsx

```diff
@@ -21,8 +21,7 @@ function useVergunningStatusLineItems(vergunning?: Vergunning) {
     ) {
       dateWorkflowActive = vergunning.dateWorkflowActive;
     }
-
-    return [
+    const statusTrain = [
```

Comment:
Dit is echt een lelijke vertaling. Statustrein is in NL al lelijk. Noem het gewoon `lineItems`

Line 1 : aanvraag
Line 2 : in behandeling
Line 3 : afgesloten